.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,273 @@
+name: "pull-request"
+
+# https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
+on:
+  pull_request:
+    types:
+      - opened
+    branches:
+      - /^master(-patch.*)?$/
+      - /^develop(-patch.*)?$/
+      - master
+      - develop
+
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  TRAVIS_BRANCH: ${{ github.ref_name }}
```

Comment:
is this still needed given that it's not a Travis anymore?

Reply:
Can we raise a task to replace the name with something more generic? @marcol13

Reply:
TRAVIS_* env vars were keeped in order to not break compatibility with the travis ci. i'll open a ticket into the current epic in order to clean everything after migration is complete